frontend/src/containers/pages/InvalidRolePage.jsx

```diff
@@ -0,0 +1,34 @@
+import BackButton from "../../components/BackButton";
+
+function InvalidRolePage() {
+  const currentUserRole = "Doctor";
+  const rolesWithAccess = ["Nurse", "Patient"];
+
+  const containerStyle = {
+    fontFamily: "Sitka Text, sans-serif",
```

Comment:
This is a custom font and you didn't import it in the rest of the application. What is its purpose here?